Aspire Budgeting/Views/AspireMasterView.swift

```diff
@@ -10,18 +10,35 @@ struct AspireMasterView: View {
   @EnvironmentObject var sheetsManager: GoogleSheetsManager
   @EnvironmentObject var appCoordinator: AppCoordinator
 
-  @State private var selectedSegment = 0
+  let tabBarItems = [TabBarItem(imageName: "rectangle.grid.1x2", title: "Dashboard"),
+                     TabBarItem(imageName: "creditcard", title: "Accounts"),
+                     TabBarItem(imageName: "arrow.up.arrow.down", title: "Transactions"),
+                     TabBarItem(imageName: "gear", title: "Settings")]
+
+  @State private var selectedTab = 0
   var body: some View {
     VStack {
       AspireNavigationBar()
         .edgesIgnoringSafeArea(.all)
         .frame(maxHeight: 65)
-      AspireSegmentedView(selectedSegment: $selectedSegment)
-      if selectedSegment == 0 {
-        DashboardView(viewModel: appCoordinator.dashboardVM)
-      } else if selectedSegment == 1 {
-        AccountBalancesView()
+      //TODO: Remove Segmented View
+//      AspireSegmentedView(selectedSegment: $selectedSegment)
+        if selectedTab == 0 {
+          DashboardView(viewModel: appCoordinator.dashboardVM)
+            .frame(height: UIScreen.main.bounds.height - 250)
+        } else if selectedTab == 1 {
+          AccountBalancesView()
+        }
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/AspireMasterView.swift#L34"></span>Limit vertical whitespace to a single empty line. Currently 3.<br />
<code>vertical_whitespace</code> <code>AspireMasterView.swift:34</code></td>
    </tr>
  </tbody>
</table>